SwaMeTest/SwaMeTest.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29306.81
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SwaMeTest", "SwaMeTest\SwaMeTest.csproj", "{4893652C-E803-4748-939B-F29AE9D08ADC}"

```

Comment:
To match the other test projects, SwameTest should be name SwaMe.Test everywhere, including directory names.